src/features/shields/select/carat.svg

```diff
@@ -1,14 +1,4 @@
-<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-  <defs>
-    <path d="M12 13.262l4.762-4.081a.75.75 0 1 1 .976 1.138l-5.25 4.5a.75.75 0 0 1-.976 0l-5.25-4.5a.75.75 0 1 1 .976-1.138L12 13.262z" id="a"/>
-  </defs>
-  <g fill="none" fill-rule="evenodd">
-    <mask id="b" fill="#fff">
-      <use xlink:href="#a"/>
-    </mask>
-    <use fill="#4A4A4A" fill-rule="nonzero" xlink:href="#a"/>
-    <g mask="url(#b)" fill="#E9E9F4">
-      <path d="M0 0h24v24H0z"/>
-    </g>
-  </g>
+<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 32 32' style='transform: rotate(270deg);width:20px;height:20px;'>
+    <path d='M22.4 29a1 1 0 0 1-.67-.25l-13.4-12a1 1 0 0 1 0-1.5l13.4-12a1 1 0 0 1 1.33 1.5L10.5 16l12.56 11.25A1 1 0 0 1 22.4 29z'
+    />
```

Comment:
A little inconsequential but if you wanted to ditch the viewbox and transforms here could swap in this: 
```
<svg width="32" height="32" xmlns="http://www.w3.org/2000/svg"><path d="M16 19.047l11.04-9.2a1.5 1.5 0 0 1 1.92 2.305l-12 10a1.5 1.5 0 0 1-1.92 0l-12-10a1.5 1.5 0 1 1 1.92-2.304l11.04 9.2z"/></svg>
```

ref: https://d.pr/i/ZVfrCo